src/App.tsx

```diff
@@ -1,24 +1,150 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoList } from './components/TodoList/TodoList';
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoError } from './components/TodoError/TodoError';
+import { Errors } from './types/Errors';
+import { Filter } from './types/Filter';
 
-const USER_ID = 0;
+const USER_ID = 10236;
+
+const preparedTodos = (todosList: Todo[], selectedFilter: Filter): Todo[] => {
+  let filteredTodos = [...todosList];
+
+  switch (selectedFilter) {
+    case 'Active':
+      filteredTodos = todosList.filter(todo => !todo.completed);
+      break;
+
+    case 'Completed':
+      filteredTodos = todosList.filter(todo => todo.completed);
+      break;
+    default:
+      break;
```

Comment:
Do not leave the `default` block empty, there you can set throwing an `error` with the text about the wrong state of the `todo`